components/n-ui/ads/index.js

```diff
@@ -13,7 +13,7 @@ window.oAds = Ads;
 function initOAds (flags, appName, adOptions) {
 	const initObj = oAdsConfig(flags, appName, adOptions);
 
-	setupAdsMetrics(flags && flags.adsDisableMetricsSampling);
+	setupAdsMetrics();
```

Comment:
I think this might be necessary to ensure we don't change the current behaviour:
```suggestion
	setupAdsMetrics(true);
```